rslib/src/storage/revlog/time_of_last_review.sql

```diff
@@ -2,5 +2,9 @@ SELECT id / 1000
 FROM revlog
 WHERE cid = $1
   AND ease BETWEEN 1 AND 4
+  AND (
+    type != 3
+    OR factor != 0
+  )
```

Comment:
If the user reviews cards early, factor will be non-zero, so the revlog entry won't be ignored. That's fine, but I just wanted to check that's what you intended here.

Reply:
Yeah. Reviews cards early should be included in the review history if the user doesn't disable `Reschedule cards based on my answers in this deck`.